contrib/apelscripts/condor_blah.sh

```diff
@@ -27,27 +27,18 @@ if [ ! -d $OUTPUT_DIR ] || [ ! -w $OUTPUT_DIR ]; then
 fi
 
 # Build the filter for the history command
-CONSTR="CompletionDate >= $yesterday && CompletionDate < $today "
+CONSTR="EnteredCurrentStatus >= $yesterday && EnteredCurrentStatus < $today && RemoteWallclockTime !=0"
 
 safe_config_val CE_HOST APEL_CE_HOST
 safe_config_val BATCH_HOST APEL_BATCH_HOST
 safe_config_val CE_ID APEL_CE_ID
 
-TZ=GMT condor_ce_history -const "$CONSTR" \
- -format "\"timestamp=%s\" " 'formatTime(CompletionDate, "%Y-%m-%d %H:%M:%S")' \
+TZ=GMT condor_history -const "$CONSTR" \
+ -format "\"timestamp=%s\" " 'formatTime(EnteredCurrentStatus, "%Y-%m-%d %H:%M:%S")' \
  -format "\"userDN=%s\" " x509userproxysubject \
  -format "\"userFQAN=%s\" " x509UserProxyFirstFQAN \
  -format "\"ceID=${CE_ID}\" " EMPTY \
- -format "\"jobID=%v_${CE_HOST}\" " 'split(GlobalJobId,"\#")[1]' \
- -format "\"lrmsID=%v_${BATCH_HOST}\" " 'split(RoutedToJobId,"\.")[0]' \
+ -format "\"jobID=%v_${CE_HOST}\" " RoutedFromJobId \
+ -format "\"lrmsID=%v_${BATCH_HOST}\" " clusterId \
```

Comment:
HTCondor-CE + HTCondor should always have a ProcId of 0 but we may just want to include it anyway just to have the full job ID (`ClusterId.ProcId`):

```suggestion
 -format "\"lrmsID=%v" clusterId \
 -format ".%v_%{BATCH_HOST}\" " ProcId \
```

Reply:
I think that should be fine, but, since lrmsID is used in  APEL client DB to JOIN blah with batch records, the equivalent change is needed on the condor_batch.sh script.

Reply:
Ok, I've got a PR in to fix the packaging (https://github.com/sjones-hep-ph-liv-ac-uk/htcondor-ce/pull/1). I'll just add a commit to fixup the condor_batch.sh, too

Reply:
So, I've added some commits to deal with the =?= operator, the procId and some minor typos. I've echoed the changes across from blah to batch system. I've tested it works the same, except procID in the lrmsId.